problemsets-mat223/modules/module1.tex

```diff
@@ -479,7 +479,24 @@
 	Let $\vec x=\xhat-\yhat$, $\vec y=3\xhat-\yhat$, and $\vec r=2\xhat+2\yhat$. Is
 	$\vec r$ a linear combination of $\vec x$ and $\vec y$?
 
-	XXX Finish
+	By definition, $\vec r$ is a linear combination of $\vec x$ and $\vec y$ if there exist scalars $a,b\in\R$ such that
+	\begin{align*}
+	    \vec r&=a\vec x+b\vec y\\
+	   \implies 2\xhat+2\yhat&=a(\xhat-\yhat)+b(3\xhat-\yhat)\\
+	   \implies 2\xhat+2\yhat&=(a+3b)\xhat+(-a-b)\yhat.
+	\end{align*}
+	Thus, it is equivalent to solve the two-equations system
```

Comment:
`Thus, it is equivalent to solve the two-equations system` -> `Rewriting the vector equation as a system of linear equations, we see we need to solve ...`